actionpack/lib/action_controller/metal/force_ssl.rb

```diff
@@ -55,10 +55,8 @@ module ClassMethods
       # You can pass any of the following options to affect the before_action callback
       # * <tt>only</tt>       - The callback should be run only for this action
       # * <tt>except</tt>     - The callback should be run for all actions except this action
-      # * <tt>if</tt>         - A symbol naming an instance method or a proc; the callback
-      #                         will be called only when it returns a true value.
-      # * <tt>unless</tt>     - A symbol naming an instance method or a proc; the callback
-      #                         will be called only when it returns a false value.
+      # * <tt>if</tt>         - A symbol naming an instance method or a proc; the callback will be called only when it returns a true value.
```

Comment:
We should avoid long lines. I think you can just add two spaces from the asterisk to the beginning of the following line and RDoc will understand.